rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1_src.json

```diff
@@ -832,8 +832,8 @@
             "minLength": 1
           },
           "note": {
-            "title": "Qualifier",
```

Comment:
all these modifications should be also done in the document-minimal jsonschema

Reply:
done